tests/conftest.py

```diff
@@ -4,6 +4,7 @@
 
 @pytest.fixture(autouse=True, scope='session')
 def cleanup_children():
+    """ reset shared memory manager for isolation, terminate any leftover processes after the test is finished """
```

Comment:
```suggestion
```

I suggest to revert the comment (shared memory is not reset anymore, terminating children is obvious from the code).